app/views/static/index.html.erb

```diff
@@ -106,7 +106,7 @@
           </div>
           <div class="text-center">
             <h3 class="text-3xl">Barcode Scanning</h3>
-            <p class="text-gray-400 mt-2">Quickly add or remove inventory with the scan of a barcode. The adjusting and transfering of inventory is also supported as well as creating your own barcodes.</p>
+            <p class="text-gray-400 mt-2">Quickly add or remove inventory with the scan of a barcode. The adjustment and transfer of inventory is also supported as well as creating your own barcodes.</p>
           </div>
```

Comment:
Oh no!  You're hitting my "CL has just enough editing training to be dangerous" button.  Which means I'm probably going to be reviewing this entire page later.

I can see why you didn't like the first version -- "The adjusting and transfering of..."  is quite awkward (and yes, there is a spelling issue there).   

It looks like  the original author was trying to keep the language really friendly -- and I agree with that approach.  "Adjustment and Transfer" is more formal, so I don't think it belongs on this page.   

Digging down into this sentence, there were other issues from the start.
1/   "Also" paired with "as well" .
2/ The "adjusting and transferring" concept belongs under Inventory Management not Barcode Scanning

I suggest this:  

Change that sentence to "Creating your own barcodes is also supported."

and then,  under Inventory Management, add:

"Adjust and transfer your inventory between locations as needed. "

Reply:
> "CL has just enough editing training to be dangerous"

I love it! I like your proposed changes and the rationale for them.